helm/overstroomik/values.yaml

```diff
@@ -0,0 +1,62 @@
+# Default values for overstroomik.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+images:
+  geoserver: deltares/overstroomik-geoserver
+  backend: deltares/overstroomik-backend
+  pullPolicy: IfNotPresent
```

Comment:
Maybe set to Always?

Reply:
Why? I propose to make a tag in github build containers in hub from the tagged versions and specify the version of containers in helm file.